src/main/java/domain/Cars.java

```diff
@@ -0,0 +1,42 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public Car findByName(String name) {
+        return cars.stream()
+                .filter(car -> car.getName().equals(name))
+                .findFirst()
+                .orElseThrow(() -> new IllegalArgumentException());
```

Comment:
<img width="640" alt="image" src="https://user-images.githubusercontent.com/59511689/217827611-071dc5a7-b53e-4ecb-9dc6-eeb718057a35.png">

메서드 참조로 변경할 수 있다고, IntelliJ에서 알려주네요!
람다의 메서드 참조에 대해 공부해보면 좋을 것 같아요~

```suggestion
                .orElseThrow(IllegalArgumentException::new);
```

Reply:
메서드 참조를 잘 몰라서 일부러 사용하지 않았는데, 이번 기회에 공부해보겠습니다!